zerver/lib/zcommand.py

```diff
@@ -54,4 +58,47 @@ def change_mode_setting(command: str, switch_command: str,
                                             switch_command='fluid-width',
                                             setting='fluid_layout_width',
                                             setting_value=False))
+    elif command == 'digress':
+        data_keys = [
+            "old_stream",
+            "old_topic",
+            "new_stream",
+            "new_topic",
+        ]
+        if not data or not all(key in data for key in data_keys):
+            raise JsonableError(_('Invalid data.'))
+
+        (old_stream, ignored_old_stream_sub) = access_stream_by_name(
```

Comment:
Use `_` for ignored variables. That is the pythonic way to do it.

Reply:
I used this style after seeing it used in other parts of the codebase ([example](https://github.com/zulip/zulip/blob/master/zerver/views/users.py#L372)). Since someone might do the same in the future, would it be worth/possible changing these instances and/or enforcing this in linters?

Reply:
Ohh! I am at fault here. We cannot use `_` here since `_` is used for text that has to be translated. That is why Steve is using those igonored_ variables too.